payment_workflow_enterprise/models/account_report_context_followup.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# © 2016 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+
+class AccountReportContextFollowup(models.TransientModel):
+
+    _inherit = 'account.report.context.followup'
+
+    @api.depends('partner_id')
+    @api.one
```

Comment:
@yasmineelmrini `@api.one` is deprecated since Odoo version 9.0. You must use a for loop.